instagram/src/App.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import './App.css';
+import dummyData from './dummy-data';
+import Posts from './components/PostContainer/posts';
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      appData: dummyData
+    }
+    dataFunction() = props => {
```

Comment:
Line 11: This is breaking your code because it needs to be outside the constructor. The syntax is also a bit off:

This line should be `dataFunction = props => {` OR `dataFunction(props) {`